src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -33,22 +42,93 @@ interface AcceptProps {
 
 interface AcceptState {
   isCommittingMutation: boolean
+  isErrorModalOpen: boolean
+  errorModalTitle: string
+  errorModalMessage: string
 }
 
+const logger = createLogger("Order/Routes/Offer/index.tsx")
+
 export class Accept extends Component<AcceptProps, AcceptState> {
   state = {
     isCommittingMutation: false,
+    isErrorModalOpen: false,
+    errorModalTitle: null,
+    errorModalMessage: null,
   }
 
   onSubmit: () => void = () => {
-    console.log("SUBMITTED!")
+    this.setState({ isCommittingMutation: true }, () => {
+      if (this.props.relay && this.props.relay.environment) {
+        commitMutation<AcceptMutation>(this.props.relay.environment, {
+          mutation: graphql`
+            mutation AcceptMutation($input: buyerAcceptOfferInput!) {
```

Comment:
#naming nit, this feels like a slightly generic-y name, and all of these queries/mutations in Reaction are global.

We've been namespacing (but not with a particularly strict rule or format AFAIK) in other areas, maybe this should be `AcceptOfferMutation`?

Here's an example of a more verbose such fragment: https://github.com/artsy/reaction/blob/988ac9d4f5cc363db4e686f8bfa83235dbae4e81/src/Apps/Collect/Components/Collection/CollectionFilterContainer.tsx#L55

In some cases, like the 'Save' button for an artwork, something genericy- like `Save_artwork`, and `SaveArtworkMutation` make sense. But, I'm also not great at naming, so just wanted to throw it out there!

Reply:
That sounds good to me! I'll make that change.